anasymod/config.py

```diff
@@ -122,12 +124,17 @@ def __init__(self, parent: EmuConfig, vivado=None, version=None,
         # set project name
         self.project_name = 'prj'
         # intermediate variables for generic Xilinx path
+        self.hints = []
+        self.hints += [lambda: os.path.join(env['VIVADO_INSTALL_PATH'], 'bin')]
         if platform in {'win32', 'cygwin'}:
-            xilinx_version_path = parent.cfg_dict['TOOLS_xilinx']
-            xilinx_version = "20" + ".".join(xilinx_version_path.split(".")[0:2]).split("-")[1]
-        # set path to vivado binary
-        self.hints = [lambda: os.path.join(env['VIVADO_INSTALL_PATH'], 'bin'),
-                      lambda: os.path.join(parent.cfg_dict['INICIO_TOOLS'], xilinx_version_path, "Vivado", xilinx_version, "bin" ),]
+            try:

```

Comment:
Note that this is now skipped if the ``xilinx_version`` determination fails to run.  That is needed because ``xilinx_version_path`` defaults to an empty string.